Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM ubuntu:18.04
+MAINTAINER flowkey
```

Comment:
@michaelknoch i don't think this belongs in the public repo

Reply:
you mean the entire Dockerfile or the commented line `MAINTAINER flowkey`